package.json

```diff
@@ -33,60 +33,58 @@
     "@bcrs-shared-components/nature-of-business": "1.0.12",
     "@bcrs-shared-components/share-structure": "2.0.4",
     "@bcrs-shared-components/staff-payment": "2.0.11",
-    "@mdi/font": "^5.5.55",
-    "@sentry/browser": "^5.23.0",
-    "@sentry/integrations": "^5.23.0",
+    "@mdi/font": "^5.9.55",
+    "@sentry/browser": "^5.30.0",
+    "@sentry/integrations": "^5.30.0",
     "array.prototype.move": "0.0.4",
-    "axios": "^0.21.1",
+    "axios": "^0.27.2",
     "core-js": "^3.6.5",
-    "http-status-codes": "^1.4.0",
-    "launchdarkly-js-client-sdk": "^2.18.1",
-    "lodash": "^4.17.20",
-    "regenerator-runtime": "^0.13.7",
+    "http-status-codes": "^2.2.0",
+    "launchdarkly-js-client-sdk": "^2.22.1",
+    "lodash": "^4.17.21",
+    "regenerator-runtime": "^0.13.9",
     "sbc-common-components": "3.0.1",
-    "uuid": "^8.3.1",
-    "vue": "^2.6.12",
+    "uuid": "^9.0.0",
+    "vue": "^2.7.10",
     "vue-affix": "^0.5.2",
     "vue-debounce-decorator": "^1.0.1",
-    "vue-router": "^3.4.3",
+    "vue-router": "^3.6.5",
     "vue-the-mask": "^0.11.1",
-    "vue2-filters": "^0.10.0",
-    "vuelidate": "^0.7.5",
-    "vuetify": "^2.3.10",
-    "vuex": "^3.5.1"
+    "vue2-filters": "^0.14.0",
+    "vuelidate": "^0.7.7",
+    "vuetify": "^2.6.10",
+    "vuex": "^3.6.2"
   },
   "devDependencies": {
-    "@types/jest": "^25.2.3",
-    "@types/lodash": "^4.14.161",
-    "@types/sinon": "^9.0.8",
+    "@types/jest": "^29.0.3",
+    "@types/lodash": "^4.14.185",
+    "@types/sinon": "^10.0.13",
     "@types/vue-the-mask": "^0.11.0",
-    "@types/vuelidate": "^0.7.13",
-    "@typescript-eslint/eslint-plugin": "^2.34.0",
-    "@typescript-eslint/parser": "^2.34.0",
-    "@vue/cli-plugin-babel": "^4.5.6",
-    "@vue/cli-plugin-eslint": "^4.5.12",
-    "@vue/cli-plugin-typescript": "^4.5.6",
-    "@vue/cli-plugin-unit-jest": "^4.5.6",
-    "@vue/cli-service": "^4.5.6",
+    "@types/vuelidate": "^0.7.15",
+    "@typescript-eslint/eslint-plugin": "^5.38.0",
+    "@typescript-eslint/parser": "^5.38.0",
+    "@vue/cli-plugin-babel": "^4.5.19",
+    "@vue/cli-plugin-eslint": "^4.5.19",
+    "@vue/cli-plugin-typescript": "^4.5.19",
+    "@vue/cli-plugin-unit-jest": "^4.5.19",
+    "@vue/cli-service": "^4.5.19",
     "@vue/eslint-config-standard": "^4.0.0",
     "@vue/eslint-config-typescript": "^4.0.0",
     "@vue/test-utils": "1.1.4",
-    "eslint": "^5.16.0",
-    "eslint-plugin-vue": "^6.2.2",
+    "eslint": "^6.8.0",
+    "eslint-plugin-vue": "^9.5.1",
     "flush-promises": "^1.0.2",
-    "sass": "^1.26.10",
-    "sass-loader": "^8.0.2",
-    "sinon": "^9.0.3",
-    "typescript": "^3.9.7",
-    "vue-class-component": "^7.2.5",
-    "vue-cli-plugin-vuetify": "^2.0.7",
-    "vue-loader-v16": "^16.0.0-beta.5.4",
```

Comment:
It appears that we didn't need `vue-class-component` or `vue-loader-v16` so I removed them. Seems to install/build/test/lint fine.

@cameron-freshworks Am I missing something?